components/formats-gpl/src/loci/formats/in/DeltavisionReader.java

```diff
@@ -174,7 +180,7 @@ public boolean isThisType(RandomAccessInputStream stream) throws IOException {
     int x = stream.readInt();
     int y = stream.readInt();
     int count = stream.readInt();
-    return x > 0 && y > 0 && count > 0;
+    return x > 0 && y > 0 && count > 0 && (x * y * count > stream.length());
```

Comment:
Shouldn't this be ```x * y * count < stream.length()```?  It would probably also be good to cast ```x```, ```y```, and ```count``` to long when multiplying, otherwise integer overflow could cause a false positive.